scripts/postgres_15_to_16_upgrade.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -e
+
+echo "Do not stop in the middle of this upgrade, wait until you see the message: Upgrade complete."
+
+echo "Stopping lemmy and all services..."
+sudo docker-compose stop
+
+echo "Make sure postgres is started..."
+sudo docker-compose up -d postgres
+echo "Waiting..."
+sleep 20s
+
+echo "Exporting the Database to 15_16.dump.sql ..."
+sudo docker-compose exec -T postgres pg_dumpall -c -U lemmy > 15_16_dump.sql
+echo "Done."
+
+echo "Stopping postgres..."
+sudo docker-compose stop postgres
+echo "Waiting..."
+sleep 20s
+
+echo "Removing the old postgres folder"
+sudo rm -rf volumes/postgres
```

Comment:
For extra safety, should this be an mv instead of an rm -rf and the script can rm at the very end which would only happen if there are no errors since it has `set -e`

Although some people might not have enough storage

Reply:
IMO the SQL backup file is enough. If that went through without failures, its much safer even than the postgres folder.